docs/Compile.rst

```diff
@@ -697,55 +704,89 @@ during a build.  If you get a failure, you may miss important errors or warnings
 due to the tiny window size and extremely limited scrollback. For that reason you
 may prefer to compile in the IDE which will always show all build output.
 
-Alternatively (or additionally), consider installing an improved Windows terminal
-such as `Cmder <https://cmder.net/>`_. Easily installed through Chocolatey with:
-``choco install cmder -y``.
 
-**Note for Cygwin/msysgit users**: It is also possible to compile DFHack from a
-Bash command line. This has three potential benefits:
 
-* When you've installed Git and are using its Bash, but haven't added Git to your path:
+From the Visual Studio IDE:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+After running the CMake generate script you will have a new folder called VC2015
+or VC2015_32, depending on the architecture you specified.
 
-  * You can load Git's Bash and as long as it can access Perl and CMake, you can
-    use it for compile without adding Git to your system path.
+Now, open the file ``dfhack.sln`` inside that folder.
+Note: You will likely be prompted to upgrade, select ``No Upgrade`` because we require this specific version
+of build tools as can be seen specified in the cmake command (ie. ``-T v140_xp``).
 
-* When you've installed Cygwin and its SSH server:
+Next, with Visual Studio open, check the build configuration drop downs. Debug does not work, so you must
+select ``RelWithDebInfo`` if you wish for debug information to be generated. Otherwise you can select
+``Release`` if you wish.
 
-  * You can now SSH in to your Windows install and compile from a remote terminal;
-    very useful if your Windows installation is a local VM on a \*nix host OS.
+Now you're ready to build. You'll find the Cmake defined targed under: ``CMakePredefinedTargets``
+You'll likely want to select ``INSTALL`` or ``ALL_BUILD`` from this folder.
 
-* In general: you can use Bash as your compilation terminal, meaning you have a decent
-  sized window, scrollback, etc.
+.. note::
 
-  * Whether you're accessing it locally as with Git's Bash, or remotely through
-    Cygwin's SSH server, this is far superior to using ``cmd.exe``.
+    If you're having trouble building, remember Visual Studio sometimes needs you
+    to rebuild or clean targets. So give that a try if you're seeing build errors.
+    Otherwise check the `FAQ`_ or `discord`_
 
-You don't need to do anything special to compile from Bash. As long as your PATHs
-are set up correctly, you can run the same generate- and build/install/package- bat
-files as detailed above.
+With the documentation
```

Comment:
This section appears to be copied directly from https://docs.dfhack.org/en/stable/docs/Documentation.html#documentation. Instead, we should link to that so that information is not duplicated. This isn't really platform-specific either.

Reply:
I was thinking along the lines of: this is simple, why send them to a new page for a simple cmake command modification